pkg/cvo/availableupdates.go

```diff
@@ -30,6 +31,7 @@ func (optr *Operator) syncAvailableUpdates(config *configv1.ClusterVersion) erro
 		usedDefaultUpstream = true
 		upstream = optr.defaultUpstreamServer
 	}
+	arch := runtime.GOARCH
```

Comment:
can I see an example of what this actually resolves to

Reply:
Here's the full list of architectures: https://github.com/golang/go/blob/release-branch.go1.10/src/go/build/syslist.go#L8

Reply:
This means we have to adhere to these strings wherever we tag builds and process these tags. Maybe we want to define our own list and map the GOARCH variants to our own?

Reply:
I had thought about doing that, but I wanted to keep this client-side code as simple as possible. We can easily change the policies server-side, in the event that a mistake is discovered. Plus, we already have to adhere to those architectures in our build system.

Reply:
> Plus, we already have to adhere to those architectures in our build system.

In that case I see no reason to put a mapping in between.

Reply:
There is a known caveat here, which is ARM 32bits. There, GOARCH is not enough to identify the architecture: https://github.com/golang/go/wiki/GoArm.

Reply:
I only want to support the ARM64 Server spec. If we want to double back and support the Raspberry PI, we can figure out some mapping scheme.